lib/js/_create/_build-new-package.js

```diff
@@ -62,25 +62,33 @@ function generateFolders(cssFolders, answers) {
 
 	if (answers.folders) {
 		for (const folder of answers.folders) {
-			folderTasks.push({
-				title: `Create folder ${answers.pkgname}/${folder}`,
-				task: () => fs.mkdir(`${filePath}/${answers.pkgname}/${folder}`, error => {
-					if (error) {
-						console.error(error);
-					} else {
-						fs.writeFile(`${filePath}/${answers.pkgname}/${folder}/.gitkeep`, '// temporary file, delete as appropriate', error => {
-							if (error) {
-								console.error(error);
-							}
-						});
-					}
-				})
-			});
+			// If the folder has sub-folders create those as well
 			if (cssFolders && cssFolders[folder]) {
 				folderTasks.push({
+					title: `Create folder ${answers.pkgname}/${folder}`,
+					task: () => {
+						// Create the parent folder
+						// Make sure it exists before processing next task
+						try {
+							fs.mkdirSync(`${filePath}/${answers.pkgname}/${folder}`);
+						} catch (error) {
```

Comment:
force the creation of the parent folder to be synchronous so it completes before the sub-folder creation